library/build.gradle

```diff
@@ -69,9 +70,9 @@ uploadArchives {
     repositories.mavenDeployer {
         beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-        repository(url: sonatypeRepo) {
```

Comment:
@serggl done. It looks like you also added fake properties in the travis yml, so I removed those as it won't be necessary